packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -218,6 +231,39 @@ const ExpandCollapseText = ({classes, isExpanded, numHidden, toggleExpansion}: E
   )
 };
 
+interface CommentViewProps {
+  comment: {
+    _id: string;
+    postId: string;
+    contents: {
+      html: string;
+      plaintextMainText: string;
+    };
+  };
+  classes: ClassesType<typeof styles>;
+}
+
+const CommentView: React.FC<CommentViewProps> = ({ comment, classes }) => {
+
+  const { PostsTooltip } = Components
+
+  const { document: post, loading, error } = useSingle({
+    collectionName: "Posts",
+    fragmentName: 'PostsPage',
+    documentId: comment.postId,
+    allowNull: true,
+  });
+
+  if (loading) return <div>Loading...</div>;
+  if (error) return <div>Error: {error.message}</div>;
+
+  return (
+    <PostsTooltip postId={comment.postId} commentId={comment._id}>
+      {comment.contents.plaintextMainText.slice(0,200)}...
```

Comment:
We have a dedicated `truncate` function for html and `truncatise` for non-html text (in this case you'd want `truncatise`)